src/org/zaproxy/zap/GuiBootstrap.java

```diff
@@ -345,15 +347,44 @@ private static void setDefaultViewLocale(Locale locale) {
      * {@link java.net.InetAddress InetAddress}) preventing some ZAP options from being correctly applied.
      */
     private void setupLookAndFeel() {
-        if (lookAndFeelSet) {
-            return;
+    	
+    	String lookAndFeelClassnameFromCommandline = System.getProperty("swing.defaultlaf");
+    	String lookAndFeelClassnameFromOptions = null;
+    	OptionsParam options = Model.getSingleton().getOptionsParam();;
+    	String lookAndFeelOptions = options.getViewParam().getLookAndFeel();
+             	  	    	
+        if(lookAndFeelClassnameFromCommandline == null) {
+        	
+        	if(lookAndFeelOptions.contains("Native")) {
+        		
+        		lookAndFeelClassnameFromOptions = UIManager.getSystemLookAndFeelClassName();
+        		
+        	} else if(lookAndFeelOptions.contains("Cross")) {
+        		
+        		lookAndFeelClassnameFromOptions = UIManager.getCrossPlatformLookAndFeelClassName();
+        	
+        	} else if(lookAndFeelOptions.contains("Default")) {
+        		
+        		lookAndFeelClassnameFromOptions = System.getProperty("swing.defaultlaf");
+        	}
```

Comment:
The blank lines in the block above could be removed.

It's probably also best if the comparisons either force a case or ignore case.

Reply:
Also, the comparisons will fail if/when the names are translated. It should be saved the classname instead of the "UI name" (and the comparisons would no longer be needed).

Reply:
I didn't considered that use case while writing this code. Thank you for pointing that out.